docs/guide/browser.md

```diff
@@ -6,6 +6,56 @@ title: Browser Mode | Guide
 
 This page provides information about the experimental browser mode feature in the Vitest API, which allows you to run your tests in the browser natively, providing access to browser globals like window and document. This feature is currently under development, and APIs may change in the future.
 
+## Installation
+
+::: code-group
+```bash [npm]
+# webdriverio/playwright should be installed based on the browsers you wish to run
+npm install -D vitest @vitest/browser webdriverio playwright
+```
+```bash [yarn]
+# webdriverio/playwright should be installed based on the browsers you wish to run
+yarn add -D vitest @vitest/browser webdriverio playwright
+```
+```bash [pnpm]
+# webdriverio/playwright should be installed based on the browsers you wish to run
+pnpm add -D vitest @vitest/browser webdriverio playwright
+```
+```bash [bun]
+# webdriverio/playwright should be installed based on the browsers you wish to run
+bun add -D vitest @vitest/browser webdriverio playwright
+```
+:::
+
+## Configuration
+
+To activate browser mode in your Vitest configuration, you can use the `--browser` flag or set the `browser.enabled` field to `true` in your Vitest configuration file. Here is an example configuration using the browser field:
+
+```ts
+export default defineConfig({
+  test: {
+    browser: {
+      enabled: true,
```

Comment:
shouldn't we add here provider?

Reply:
AFAIU from [this code](https://github.com/vitest-dev/vitest/blob/main/packages/vitest/src/types/browser.ts#L46), the default is `webdriverio` if not specified and `chrome` is a part of `webdriverio` so this should work but I'm definitely open to adding it if you think it will help :)